django/test/testcases.py

```diff
@@ -889,18 +908,23 @@ def assertInHTML(self, needle, haystack, count=None, msg_prefix=""):
         haystack = assert_and_parse_html(
             self, haystack, None, "Second argument is not valid HTML:"
         )
+        if msg_prefix:
+            msg_prefix += ": "
```

Comment:
This isn’t related to the other changes here. I can see why this change would be worthwhile for consistency with other methods, though it seems there are other methods that are missing this, so perhaps worth addressing in another PR that changes this everywhere. 

If we do want to proceed with this as-is, I guess this does need to be acknowledged separately in the release notes.

Reply:
I think we can keep this one in this PR as is and create a separate ticket for addressing this change in all the methods that do not have this.

Reply:
Sounds good to me, you’ll need to add an entry in the release notes for this as well then.

Reply:
Done 👍

Reply:
Please remove it for this patch as it's not related with the ticket. We don't mix different features/bugfixes/cleanups in a single commit.

Reply:
I moved it to the #17332.